src/components/ha-markdown.html

```diff
@@ -1,65 +1,59 @@
-<link rel='import' href='../../bower_components/polymer/polymer-element.html'>
-
 <link rel='import' href='../util/hass-mixins.html'>
 
 <script>
-class HaMarkdown extends window.hassMixins.EventsMixin(Polymer.Element) {
+class HaMarkdown extends window.hassMixins.EventsMixin(HTMLElement) {
   static get is() { return 'ha-markdown'; }
 
-  static get properties() {
-    return {
-      content: {
-        type: String,
-      },
+  connectedCallback() {
+    // 0 = not loaded, 1 = success, 2 = error
+    this._scriptLoaded = 0;
+    this._renderScheduled = false;
+    this._resize = () => this.fire('iron-resize');
 
-      scriptLoaded: {
-        type: Number,
-        // 0 = not loaded, 1 = success, 2 = error
-        value: 0,
-      },
-    };
+    Polymer.importHref(
+      '/static/pagedown-js.html',
+      () => { this._scriptLoaded = 1; this._render(); },
+      () => { this._scriptLoaded = 2; this._render(); },
+    );
   }
 
-  static get observers() {
-    return [
-      'updateContent(content, scriptLoaded)',
-    ];
+  set content(value) {
```

Comment:
Does attribute binding work without polymer?

Reply:
No. PropertyAccessors mixin would take care of that.

Reply:
Actually, it looks like it has methods for it ([like this](https://github.com/Polymer/polymer/blob/master/lib/mixins/property-accessors.html#L261-L274)) but doesn't seem to be hooked up.

Reply:
Ah, it's defined in PropertyAccessors but it's another mixin that takes care of it: https://github.com/Polymer/polymer/blob/master/lib/mixins/property-effects.html

Reply:
But those mixins are not used here. And the users of HaMarkdown set the content in an attribute. So what will cause the attribute to propagate to a property?

Reply:
Polymer is actually setting it as a property in their templating engine.